### DISCUSSION TITLE
Automatic sidebar display when starting brackets

### DISCUSSION THREAD

**Discussion by Ralf1234567890** [brackets-cont/brackets#109] [Q&A] [answered]
Hi there,
when starting brackets, the sidebar is always hidden. How can I always show this automatically when starting?
Many Thanks.

**Comment by charlypa** [comment, marked as answer]
@Ralf1234567890  check out below on how to do this
![sidebar](https://user-images.githubusercontent.com/1525027/147531754-f78e448c-a519-4a46-aebc-ab02fc60f614.gif)

**Reply by Ralf1234567890** [reply]
Yes, of course I did. But after closing and restarting the sidebar is hidden again.
Many Thanks.

**Reply by charlypa** [reply]
@Ralf1234567890 Could you share your OS and version of brackets ?

**Reply by Ralf1234567890** [reply]
Solved: I completely uninstalled and reinstalled Brackets. Now it's working.
Windows 10, Brackets Release 2.0
